src/data/mv_polynomial/basic.lean

```diff
@@ -1264,3 +1264,47 @@ end aeval_tower
 end comm_semiring
 
 end mv_polynomial
+
+namespace subsemiring
```

Comment:
Does this generalize easily to `subring_class`?

Reply:
Yes, and done